sirepo/simulation_db.py

```diff
@@ -994,7 +981,7 @@ def _pkg_relative_path_static(file_dir, file_name):
     Returns:
         str: full relative path of the file
     """
-    return '/' + RESOURCE_FOLDER.bestrelpath(static_file_path(file_dir, file_name))
+    return '/' + sirepo.resource.static(file_dir, file_name, relpath=True)
```

Comment:
There should be documentation about how this works, because it is going to have to be found later when the client requests the file. It works, because the search path is the same.

Reply:
6e28eb52